examples/js/objects/Reflector.js

```diff
@@ -172,19 +172,9 @@ THREE.Reflector = function ( geometry, options ) {
 
 		// Restore viewport
 
-		var bounds = camera.bounds;
+		if ( camera.viewport !== undefined ) {
```

Comment:
Sorry, one more thing. Can these checks be `if ( camera.isArrayCamera ) {`.

Reply:
@mrdoob done!